src/main/java/net/imagej/legacy/ImageJ2Options.java

```diff
@@ -237,4 +243,29 @@ private int parseLogLevel(String level) {
 		
 		return LogService.WARN;
 	}
+
+	/**
+	 * Parses a log level from a {@code int}
+	 * 
+	 * FIXME Could be removed if log level is defined as enumeration
```

Comment:
Please mark this as `TODO` rather than `FIXME`. From the [ImageJ coding style page](http://imagej.net/Coding_style#Javadoc_and_comments):

> * For code (or lack thereof) that is considered wrong or broken, and in need of repair, we add a "FIXME" comment with the relevant developer's initials, to serve as a reminder to address it as soon as time allows.
> * When additional work is needed somewhere, but not urgently, we add a "TODO" comment marking it.